core/app/assets/js/molecules/select-file.js

```diff
@@ -0,0 +1,49 @@
+import { useRef } from 'react';
```

Comment:
Move all `input` components to `core/app/assets/js/ui/inputs`

Reply:
We would like to have a discussion about this issue and files architecture in general, after the release.
Let's leave it as is for now...